### ISSUE TITLE
Sentence splitting

### ISSUE THREAD

**Issue by mfilipov** [honnibal/spaCy#67]
Python version: 3.4.3, latest spaCy (0.85), redownloaded all data (python -m spacy.en.download all).

It looks like there's something wrong with sentence splitting. Example: 
```
pipeline = spacy.en.English()

tokens = pipeline("The Germans have their Faust; but Faust is a tragedy with a cosmic philosophic theme.")

root_inds = [ind for ind, token in enumerate(tokens) if token.dep_ == "ROOT"]

```
`root_inds` has two elements - corresponding sentences are "The Germans have their Faust; but" and "Faust is a tragedy with a cosmic philosophical theme.".

I guess this is related to the ROOT bug https://github.com/honnibal/spaCy/issues/57.

**Comment by honnibal** [comment]
Thanks — working on this.

**Comment by honnibal** [comment]
Just released version 0.86.

Your example now parses correctly, and accuracy is up on aggregate. Further improvements to sentence boundary detection accuracy should be forth-coming.

Please keep reporting prominent failures as they occur.
